src/index.js

```diff
@@ -57,6 +60,7 @@ function setup(webContents, { socketTimeout, socketKeepAliveDelay } = {}) {
       );
       // Notify the renderer process that we are listening for notifications
       webContents.send(NOTIFICATION_SERVICE_STARTED, credentials.fcm.token);
+      started = true
```

Comment:
Previously we marked it started before seeing if this succeeded or not. Which would mean that subsequent retrys wouldn't work.